### DISCUSSION TITLE
Cursor size changes

### DISCUSSION THREAD

**Discussion by darrell-roberts** [Relm4/Relm4#395] [Q&A]
Hi,

I noticed that when run my relm4 app the cursor size shrinks when my mouse enters my app. I'm running Ubuntu with Gnome 42.5 and Wayland. I'm have my display configured with 200% scale. Is there something I need to do to ensure a Gtk4 app uses the right cursor size for my display setting?

**Comment by AaronErhardt** [comment]
AFAIK this should be done by GTK4 automatically. I think there's nothing Relm4 or even gtk-rs could do to fix this. But you could ask in the [gtk-rs chat room](https://matrix.to/#/#rust:gnome.org) where people are more experienced with GTK in general.
